<!DOCTYPE html>
<html >
<head>
  <!-- Site made with Mobirise Website Builder v4.9.3, https://mobirise.com -->
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Mobirise v4.9.3, mobirise.com">
  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
  <link rel="shortcut icon" href="assets/images/dna-grey-122x122.png" type="image/x-icon">
  <meta name="description" content="Site Maker Description">
  <title>Publications</title>
  <link rel="stylesheet" href="assets/tether/tether.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap-grid.min.css">
  <link rel="stylesheet" href="assets/bootstrap/css/bootstrap-reboot.min.css">
  <link rel="stylesheet" href="assets/dropdown/css/style.css">
  <link rel="stylesheet" href="assets/theme/css/style.css">
  <link rel="stylesheet" href="assets/mobirise/css/mbr-additional.css" type="text/css">
  
  
  
</head>
<body>
  <section class="menu cid-r0likYJYKI" once="menu" id="menu1-i">

    

    <nav class="navbar navbar-expand beta-menu navbar-dropdown align-items-center navbar-fixed-top navbar-toggleable-sm">
        <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
            <div class="hamburger">
                <span></span>
                <span></span>
                <span></span>
                <span></span>
            </div>
        </button>
        <div class="menu-logo">
            <div class="navbar-brand">
                <span class="navbar-logo">
                    <a href="page1.html">
                         <img src="assets/images/dna-grey-122x122.png" alt="Mobirise" title="" style="height: 3.8rem;">
                    </a>
                </span>
                <span class="navbar-caption-wrap"><a class="navbar-caption text-white display-5" href="assets/files/Krampis%20-%20CV%20-%20Jan-%202019.pdf">Konstantinos Krampis, PhD</a></span>
            </div>
        </div>
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
            <ul class="navbar-nav nav-dropdown nav-right" data-app-modern-menu="true"><li class="nav-item">
                    <a class="nav-link link text-white display-4" href="index.html">Home</a>
                </li><li class="nav-item"><a class="nav-link link text-white display-4" href="page7.html">Research</a></li><li class="nav-item"><a class="nav-link link text-white display-4" href="page2.html">Lab Members</a></li><li class="nav-item"><a class="nav-link link text-white display-4" href="page3.html">Publications</a></li><li class="nav-item"><a class="nav-link link text-white display-4" href="page9.html">Funding</a></li><li class="nav-item"><a class="nav-link link text-white display-4" href="page11.html">Awards</a></li><li class="nav-item dropdown"><a class="nav-link link text-white display-4" href="page4.html" aria-expanded="false">Teaching</a></li><li class="nav-item"><a class="nav-link link text-white display-4" href="page5.html">Contact</a></li></ul>
            
        </div>
    </nav>
</section>

<section class="engine"><a href="https://mobirise.info/q">responsive website templates</a></section><section class="mbr-section article content9 cid-r0mYzN2DeI" id="content9-1h">
    
     

    <div class="container">
        <div class="inner-container" style="width: 100%;">
            <hr class="line" style="width: 25%;">
            <div class="section-text align-center mbr-fonts-style display-7"><strong>PUBLICATIONS</strong><br><strong><br></strong><div><em><strong>High-throughput bioinformatics infrastructures for cluster and cloud computing platforms</strong></em></div><div><br></div><div>1. Kim B., Ali T., Dong C., Laungani B., Wultsch C., Lijeron C. and Krampis K. (2019). J. of Computational Biology. &nbsp;26:3. “miCloud: a plug and play, on-premises bioinformatics cloud, providing seamless integration with Illumina genome sequencers”.</div><br><div>2. Ali T, Kim B, Lijeron C, Dong C, Wultsch C and Krampis K (2018) J Comput Sci Syst Biol 11:  195-199. "Implementation of a Reproducible,
&nbsp;<span style="font-size: 1rem;">Accessible and Transparent RNA-seq Bioinformatics Pipeline within the Galaxy Platform".&nbsp;</span></div><div><br></div><div>3. Dong C, Lee R, Sayad J and Krampis K. (2018) Advancements Bioequiv Availab. 1(2). ABB.000508.2018. &nbsp;"Bioinformatics Programming for Bioavailability Analysis of
&nbsp;<span style="font-size: 1rem;">Sequence Patterns in Public Genomic Databases".&nbsp;</span></div><br><div><span style="font-size: 1rem;">4.Alterovitz G., Dean D.A., Carole Goble C, Crusoe MR. … Krampis K., et al. (2018). PLoS Biol. &nbsp;</span><span style="font-size: 1rem;">31;16(12). “Enabling Precision Medicine via standard communication of NGS provenance, analysis, and results”.</span></div><div><br></div><div>5.&nbsp;<a href="https://f1000research.com/articles/6-2162/v1" class="text-primary">Jensen T.L., Frasketi M., Conway K., Villarroel L., Hill H., Krampis K. and Goll J.B. (2018). F1000Research, 6, 2162. “RSEQREP: RNA-Seq Reports, an open-source cloud-enabled framework for reproducible RNA-Seq data processing, analysis, and result reporting”.</a></div><div><br></div><div>6.&nbsp;<a href="https://peerj.com/preprints/3385/">Ali T., Kim B., Lijeron C., Ogunwobi O.O., Mazumder R. and Krampis K. (2017). PeerJ , 5, p.e3385v1. “TED toolkit: a comprehensive approach for convenient transcriptomic profiling as a clinically oriented application”.</a></div><div><br></div><div>7.&nbsp;<a href="https://academic.oup.com/gigascience/article/6/8/1/3894236">Kim B., Ali T., Lijeron C. and Krampis K. (2017). Gigascience. 6(8), 1–7. “Bio-Docklets: Virtualization Containers for Portable, Scalable NGS Data Analysis”.</a></div><div><br></div><div>8.&nbsp;<a href="https://academic.oup.com/bioinformatics/article/32/13/2050/1743980">Kim B., Ali T., Hosmer S. and Krampis K. (2016). Bioinformatics, 10, 1093. “Visual Omics Explorer (VOE): a Cross-Platform Portal for Interactive Data Visualization”.</a></div><div><br></div><div>9.&nbsp;<a href="https://ieeexplore.ieee.org/document/7160269/?reload=true">Afgan E., Krampis K., Goonasekera N., Skala K. and Taylor J. (2015). IEEE-MIPRO, 38, 223-228. “Building and provisioning bioinformatics environments on public and private clouds”.</a></div><div><br></div><div>10.&nbsp;<a href="https://content.sciendo.com/view/journals/mngs/open-issue/article-10.1515-mngs-2015-0003.xml">Krampis K. and Wultsch C. (2015). Methods in Next Generation Sequencing, 2(1). “A Review of Cloud Computing Bioinformatics Solutions for Next-Gen Sequencing Data Analysis and Research”.</a></div><div><br></div><div>11.&nbsp;<a href="https://f1000research.com/articles/4-20/v1">Kumari P., Mazumder R., Simonyan V. and Krampis K. (2015). F1000 Research, 4, 20. “Advantages of distributed and parallel algorithms that leverage Cloud Computing platforms for large-scale genome assembly”.</a></div><div><br></div><div>12.&nbsp;<a href="https://f1000research.com/articles/4-20/v1">Krishnakumar V., Hanlon M.R., Contrino S., Ferlanti E.S., Krampis K…and Town C.D. (2014). Nucleic Acids Research, 28(43), 1003-1009. “Araport: the Arabidopsis Information Portal”.</a></div><div><br></div><div>13.&nbsp;<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-42">Krampis K., Booth T., Chapman B., Tiwari B., Field D. and Nelson K.E. (2012). BMC Bioinformatics, 13, 42. “Cloud Biolinux: pre-configured and on-demand computing for the genomics community”.</a></div><div><br></div><br><div><br></div><div><em><strong>Bioinformatics for Next Generation Sequencing (NGS) clinical and personal genomics</strong></em></div><div><br></div><div>1. King CH, Desai H, Sylvetsky A, LoTempio J ... Krampis K, et al. (2019). <em>&nbsp;In preparation for PLoS One</em>. “A new standard for defining a baseline of&nbsp;<span style="font-size: 1rem;">healthy human gut microbiomes”</span></div><br><div>
</div><div>2. Deutsch DR, Euler CW, Utter B, Schuch R, Verratti K, Krampis K, Beyer W and Fischetti VA. (2019). <em>In preparation for Nature Sci. Reports.&nbsp;</em>Alteration of&nbsp;<span style="font-size: 1rem;">Canonical B. anthracis Phenotypes After Exposure to a B. cereus Bacteriophage”</span><span style="font-size: 1rem;">.</span></div><br><div>3. Brown S.M., Hao Y., Chen H., Laungani B.P., Ali T.A., Dong C., Lijeron C., Kim B., Krampis K. and Pei Z. (2019). <em>Gigascience - Accepted, In Press.</em> "Fast functional annotation of metagenomic shotgun data by DNA alignment to a microbial gene catalog".</div><div><br></div><div>4.&nbsp;<a href="https://www.sciencedirect.com/science/article/pii/S2352340917300161?via%3Dihub">Das D.K., Ali T., Krampis K. and Ogunwobi O. (2017). Data in Brief, 11, 131-35. “Fibronectin and androgen receptor expression data in prostate cancer obtained from a RNA-sequencing bioinformatics analysis”.</a></div><div><br></div><div>5.&nbsp;<a href="https://www.sciencedirect.com/science/article/pii/S0014482716303135?via%3Dihub">Das D.K., Naidoo M., Ilboudo A., Park J.Y., Ali T., Krampis K., Robinson B.D., Osborne J.R. and Ogunwobi O. (2016). Experimental Cell Research, 348(2), 190-200. “miR-1207-3p regulates the androgen receptor in prostate cancer via FNDC1/fibronectin”.</a></div><div><br></div><div>6.&nbsp;<a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0141712">Bubnell J., Jamet S., Tomoiaga D., D’Hulst C., Krampis K. and Feinstein P. (2015). PloS ONE, 10(10), p.e0141712. “In Vitro Mutational and Bioinformatics Analysis of the M71 Odorant Receptor and Its Superfamily”.</a></div><div><br></div><div>7.&nbsp;<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-28">Cole C., Krampis K., Karagiannis K., Almeida J.S., Faison W.J., Motwani M., Wan Q., Golikov A., Pan Y., Simonyan V. and Mazumder R. (2014). BMC Bioinformatics, 15, 28. “Non-synonymous variations in cancer and their effects on the human proteome: workflow for NGS data biocuration and proteome-wide analysis of TCGA data.”</a></div><div><br></div><div>8.&nbsp;<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-918">Shamsaddini A., Yang P., Johnson E., Krampis K., Simonyan V. and Mazumder M. (2014). BMC Genomics, 15, 918. “Census-based rapid and accurate metagenome taxonomic profiling”.</a></div><div><br></div><br><div><br></div><div><em><strong>Interdisciplinary research and novel applications of genomic sequencing and bioinformatics</strong></em></div><div><br></div><div>1. Wultsch C., Perkins S.L., Kelly M.J., Waits L.P., Quigley H., Amato G., and Krampis K. (2019)<em> In preparation for Nature, Ecology and Evolution. </em>“A new perspective in carnivore ecology and conservation: comparative gut microbiome composition, diversity, and functionality in sympatric Neotropical jaguars and pumas”.</div><br><div>2. Wultsch C, Benton A, Sosa Medina G, McNitt D, Alonso R, Kelly MJ and Krampis K. (2019)<em> In preparation for Microbiome.</em> “Bacterial gut, skin
&nbsp;<span style="font-size: 1rem;">and oral microbiomes of free-ranging bobcats (Lynx rufus)”</span></div><div><span style="font-size: 1rem;"><br></span></div><div>3. Wultsch C, DeBarba M, Kelly MJ, Waits LP and Krampis K. (2019) <em>In preparation for Molecular Ecology.</em> “Resource partitioning and niche overlap in a
&nbsp;<span style="font-size: 1rem;">Neotropical felid community examined by DNA metabarcoding”.</span></div><div><br></div><div>4.&nbsp;<a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.00803/full">Ingala M.R., Simmons N.B., Wultsch C., Krampis K., Speer K.A., and S.L. Perkins. (2018). Frontiers in Microbiology, 1;9:803. “Comparing Microbiome Sampling Methods in a Wild Mammal: Fecal and Intestinal Samples Record Different Signals of Host Ecology, Evolution”.</a></div><div><br></div><div>5.&nbsp;<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1334-6">Prince S.J., Song L., Qiu D., dos Santos J.V.M., Chai C., Joshi T., Patil G., Valliyodan B., Vuong T.D., Murphy M. and Krampis, K. (2015). BMC Genomics, 16(1), 132. “Genetic variants in root architecture-related genes in a Glycine soja accession, a potential resource to improve cultivated soybean”.</a></div><div><br></div><div>6.&nbsp;<a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/3/1/23">Wang H., Waller L., Tripathy S., St. Martin S.K., Zhou L., Krampis K. et al. (2010). The Plant Genome 3(1), 23-40. “Analysis of genes underlying soybean QTLs conferring partial resistance to </a><em><a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/3/1/23">P. sojae</a></em><a href="https://dl.sciencesocieties.org/publications/tpg/abstracts/3/1/23">”.</a></div><div><br></div><div>7.&nbsp;<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-10-49">Zhou L., Mideros S.X., Bao L., Hanlon R., Arredondo F., Tripathy S., Krampis K. et al. (2009). BMC Genomics, 26(10), 49. "Infection and genotype remodel the entire Soybean transcriptome".</a></div><div><br></div><div>8. Tyler B.M., Tripathy S., Kale S.D., Zhou L., Ferreira A., Dou D., Arredondo F.D., Mideros S.X., Bao L., Krampis K. and Jerauld, A. (2010). Phytopathology 99 (6), S164. “Comparative and functional genomics of oomycete infection”.</div><div><br></div><div>9.&nbsp;<a href="https://link.springer.com/article/10.1007/s11103-008-9322-8">Prasinos C., Haralampidis K., Milioni D., Samakovli D., Krampis K. and Hatzopoulos P. (2008). Plant Molecular Biology, 67(4), 323. “Complexity of Hsp90 in organelle targeting”.</a></div><div><br></div><div>10.&nbsp;<a href="https://link.springer.com/article/10.1007/s11103-008-9322-8">Wang H., Berry S., St. Martin S.K., Zhou L., Krampis K. et al. (2008). Phytopathology 98 (6), S165. “Allele mining for genes associated with partial resistance to <em>Phytophthora sojae</em> in soybean”.</a></div><div><br></div><div>11.&nbsp;<a href="https://apsjournals.apsnet.org/doi/10.1094/MPMI-19-1329">Krampis K., Tyler B.M. and Boore J.L. (2006) Molecular Plant Microbe Interactions, 19(12), 1329-1336. "Extensive variation in nuclear mitochondrial DNA content between the genomes of </a><em><a href="https://apsjournals.apsnet.org/doi/10.1094/MPMI-19-1329">P. sojae</a></em><a href="https://apsjournals.apsnet.org/doi/10.1094/MPMI-19-1329"> and </a><em><a href="https://apsjournals.apsnet.org/doi/10.1094/MPMI-19-1329">P. ramorum</a></em><a href="https://apsjournals.apsnet.org/doi/10.1094/MPMI-19-1329">".</a></div><div><br></div><div>12.&nbsp;<a href="http://science.sciencemag.org/content/313/5791/1261.long">Tyler B.M., Tripathy S…, Krampis K., et al. (2006) Science 313(5791), 1261-1266. "Phytophthora genome sequences uncover evolutionary origins and mechanisms of pathogenesis".</a></div><div><br></div><div>13.&nbsp;<a href="https://academic.oup.com/jxb/article/56/412/633/580242">Prasinos C., Krampis K., Samakovli D. and Hatzopoulos P. (2005). Journal of Experimental Botany, 56(412), 633-644. “Tight regulation of expression of two Arabidopsis cytosolic Hsp90 genes during embryo development”.</a></div><div><br></div><br><div><br></div><div><em><strong>Bibliography - Monographs</strong></em></div><div><br></div><div>1. Krampis K. “Cloud Computing for Bioinformatics: Next-Gen Sequencing and Large-Scale Data Analytics” Monograph to be published in 2019, contract signed with Cambridge University Press, Cambridge, UK.</div><div><br></div><br><div><br></div><div><strong><em>Bibliography - Book Chapters</em></strong></div><div><br></div><div>1. Krampis K., Efstratios E. and Brown S.T. (2015, 2017). “Cloud-Based Next-Generation Sequencing Informatics”. Book Chapter in Next-Generation DNA Sequencing Informatics (pp. 361-370), First &amp; Second Edition. Cold Spring Harbor Press, NY.</div><div><br></div><div>2. Rusch D.B., Miller J., Krampis K., Tovchigrechko A., Sutton G., Yooseph S. and Nelson K.E. (2014). “Bioinformatics for Genomes and Metagenomes in Ecology Studies”. Book Chapter in Infectious Microecology (pp. 203-226). Springer GmbH &amp; Co, Berlin, Germany.</div><div><br></div><div>3. Goll J.B, Szpakowski S., Krampis K. and Nelson K.E. (2014). “Next-generation sequencing, metagenomes, and the human microbiome”. Book chapter in Bioinformatics and Data Analysis in Microbiology (Chpt. 7). Caister Academic Press, Norfolk, U.K.</div><div><br></div><div>4. Nelson K.E., Madupu R., Szpakowski S., Goll J.B., Krampis K. and Methe B.A. (2014). “Next-generation sequencing, metagenomes, and the human microbiome”. Book chapter in Next-generation Sequencing: Current Technologies and Applications (pp. 141-155). Caister Academic Press, Norfolk, U.K.</div><div>Quirino B.F., Barreto C.C., Pappas G.J., Zengler K., Krampis K. and Krüger R.H. (2013). “Genomes and Post-Genome Technology” Book Chapter in Prokaryotes (pp. 329-344). Springer GmbH &amp; Co., Berlin, Germany.</div><div><br></div><div>5. Madupu R., Rogers Yu-H., Rusch D., Miller J., Krampis K. and Nelson K.E. (2012). “Microbiomes”. Book Chapter in Encyclopedia of Molecular Cell Biology and Molecular Medicine. Wiley-VCH Verlag GmbH &amp; Co., Weinheim, Germany.</div><div><br></div><div>6. Miller J., Rusch D., Krampis K., Tovichgrechko A., Sutton G., Yooseph S. and Nelson K.E. (2012). “Bioinformatics for Genomes and Metagenomes in Microbial Ecology Studies”. Book Chapter in Encyclopedia of Molecular Cell Biology and Molecular Medicine. Wiley-VCH Verlag GmbH &amp; Co., Weinheim, Germany.</div><div><br></div><div>7. Tyler B.M., Jiang R.H., Zhou L., Tripathy S., Dou D., Torto-Alalibo T., Li H., Mao Y., Liu B., Vega-Sanchez M., Mideros S.X., Hanlon R., Smith B.M., Krampis K., Ye K., Martin S.S., Dorrance A.E., Hoeschele I. and Sagahi Maroof M.A. (2008). “Functional genomics and bioinformatics of the Phytophthora sojae soybean interaction”. Book Chapter in Genomics of Disease (pp. 67-78). Springer GmbH &amp; Co, NY.</div><div><br></div></div>
            <hr class="line" style="width: 25%;">
        </div>
        </div>
</section>


  <script src="assets/web/assets/jquery/jquery.min.js"></script>
  <script src="assets/popper/popper.min.js"></script>
  <script src="assets/tether/tether.min.js"></script>
  <script src="assets/bootstrap/js/bootstrap.min.js"></script>
  <script src="assets/smoothscroll/smooth-scroll.js"></script>
  <script src="assets/dropdown/js/script.min.js"></script>
  <script src="assets/touchswipe/jquery.touch-swipe.min.js"></script>
  <script src="assets/theme/js/script.js"></script>
  
  
</body>
</html>